script/install-repos.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+git clone git@github.com:department-of-veterans-affairs/vets-website.git
+git clone git@github.com:department-of-veterans-affairs/vagov-content.git
```

Comment:
These are being cloned to a location inside vets-website, but they should be cloned as siblings to vets-website. 

vagov-content is the only one that is required for the build process, it's possible we're trying to do too much by installing all of them.

Reply:
Ok. I've changed the script to clone to parent dir, and commented out the other repos until we confirm they are not needed.